libraries/core_libs/consensus/include/vote_manager/vote_manager.hpp

```diff
@@ -27,7 +27,7 @@ class NextVotesManager {
   /**
    * @brief Clear previous PBFT round next voting type votes
    */
-  void clear();
+  void clearVotes();
```

Comment:
Should this be called clearNextVotes() if only clears next votes?

Reply:
coul be but as it is NextVotesManager it seemed to me clear that it does not have any other votes than next votes